atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -185,7 +195,8 @@ def choose_cluster(self, platform):
             raise RuntimeError('No clusters found for platform {}!'
                                .format(platform))
 
-        selected = min(clusters, key=lambda c: c.load)
+        reachable_clusters = filter(lambda x: x.load != self.UNREACHABLE_CLUSTER_LOAD, clusters)
```

Comment:
Good idea, added an additional test case and exception when all clusters are unreachable